docs/api/API_Doc.md

```diff
@@ -0,0 +1,21 @@
+# API Requests for Data Exchange
+
+This document provides about API requests for the Data Exchange.
+
+## Postman
+
+There is a [postman collection](./postman) containing information on how to exchange data between provider and consumer.
+
+⚠️ _Change de webhook.site variable value to your respective one._
```

Comment:
Is this still a valid statement after switching to the simple data backend?

Reply:
You are right, going to remove it, and answering the other question, with the Simple Data Backend the Data Exchange can be performed, it works fine. 😃